### PR TITLE
174 login email case sensitive

### PR DESCRIPTION
**Pull request by nasief**

### CODE REVIEW

**Comment by nasief** [comment]
[(1) Login - Bug - Email is case sensitive [Cycle8]](https://trello.com/c/RS7gM3sO/174-1-login-bug-email-is-case-sensitive-cycle8)

File: accountsplus/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '1.3.1'
+__version__ = '1.6.0'
```
Comment (waseem-omar) [inline accountsplus/__init__.py]: @nasief Why `1.6.0`?

File: accountsplus/admin.py [hunk 2]
```diff
@@ -20,15 +20,23 @@
...
 import signals
 import models
+import forms
 
 
 sensitive_post_parameters_m = django.utils.decorators.method_decorator(
     django.views.decorators.debug.sensitive_post_parameters())
 
 
+class CustomAdminSite(AdminSite):
```
Comment (waseem-omar) [inline accountsplus/admin.py]: @nasief Do we need this anymore?

Comment (nasief) [reply accountsplus/admin.py]: @waseem-omar no we don't, I will remove it.

File: accountsplus/__init__.py (hunk 1 above)
Comment (nasief) [reply accountsplus/__init__.py]: @waseem-omar I was testing something and I thought there was an issue with my version. It should not have been included. Will remove it

**Comment by waseem-omar** [comment]
[OSS - django-users-plus - Merge email case sensitive changes to master](https://trello.com/c/vtMw6hD4/253-oss-django-users-plus-merge-email-case-sensitive-changes-to-master)

File: accountsplus/forms.py [hunk 3]
```diff
@@ -17,3 +19,15 @@ def clean_username(self):
         if not User.objects.filter(email=username).exists():
             raise django.forms.ValidationError("Username does not belong to a registered user")
         return username
+
+
+class EmailBasedAuthenticationForm(AuthenticationForm):
+
+    def clean_username(self):
+        return self.data['username'].lower()
```
Comment (awartani) [inline accountsplus/forms.py]: @nasief maybe you need to trim the spaces around it as well.

**Review by awartani — APPROVED** [review]

**Review by awartani — APPROVED** [review]

**Pull request closed without merging**